content/develop/connect/clients/comm-supp-clients.md

```diff
@@ -0,0 +1,25 @@
+---
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: Connect to a Redis database with other languages
+linkTitle: Community-supported clients
+title: Community-supported clients
+weight: 7
+---
+
+The table below lists the recommended client libraries for languages that
+are not supported by Redis directly.
+
+| Language | Client name | Github | Docs |
+| :-- | :-- | :-- | :-- |
+| C | hiredis | https://github.com/redis/hiredis | |
```

Comment:
Even though the Docs page for hiredis is the same as the repo, I'd include it for completeness.

```suggestion
| C | hiredis | https://github.com/redis/hiredis | https://github.com/redis/hiredis |
```